resources/frameworks.yaml

```diff
@@ -109,3 +109,11 @@ GAMA:
 autoxgboost:
   version: '0.0.0.9000'
   project: https://github.com/ja-thomas/autoxgboost
+
+mljarsupervised:
+  version: '0.5.5'
+  project: https://github.com/mljar/mljar-supervised
+  params:
+    mode: "Compete"   # set mode for Compete, default mode is Explain
```

Comment:
@pplonski for the "official" openml benchmark, we currently require that all AutoML frameworks use their default configuration outside the time/cores/metrics constraints imposed by benchmark app, and the parameters defining the ML problem (+dataset).
This `mode` parameter should then be commented out.

@PGijsbers what do you think about this kind of parameter? Given the evolution of focus of many ML frameworks, we see more and more presets/high-level configs like that (AutoGluon has a similar approach, I'm thinking about sth similar for H2OAutoML...).
I sometimes find it weird that we refuse this kind of param when we benchmark only on prediction performance.